pkg/kubernetes/views/add-member-role-dialog.html

```diff
@@ -2,7 +2,7 @@
     <div class="modal-header">
         <h4 class="modal-title">
           <span translate>Add Member</span>
-          <a tabindex="0" role="button" popover-trigger="focus" popover-placement="bottom"
+          <a tabindex="0" tabindex="0" role="button" popover-trigger="focus" popover-placement="bottom"
```

Comment:
```suggestion
          <a tabindex="0" role="button" popover-trigger="focus" popover-placement="bottom"
```

Again, should be a `<button>`, really.